### PR TITLE
[Price Impact] Fix broken warnings on adjusted market

### PR DESCRIPTION
**Pull request by W3stside**
Fixes broken warnings when adjusted smart quote markets come into play

Makes sure to use correct price and absolute token pair in price impact calculation

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
   * **🎩 [Dapp](https://pr1557--dexreact.review.gnosisdev.com)**: Testing web app
   * **📚 [Storybook](https://pr1557--dexreact.review.gnosisdev.com/storybook/)**: Component stories

**Review by Velenir — APPROVED** [review]

File: src/components/TradeWidget/index.tsx [hunk 1]
```diff
@@ -685,7 +685,8 @@ const TradeWidget: React.FC<TradeWidgetProps> = ({
             sellToken={sellToken}
```
Comment (anxolin) [inline src/components/TradeWidget/index.tsx:685]: Logic seems correct, but this component already receives the "sell token" and the "quote token", I understand this logic responsibility lies inside the `<Price Suggestions>`, isn't it?

I'm referring to:
```js
            limitPrice={sellToken === quoteToken ? priceValue : priceInverseValue}
```

Comment (anxolin) [reply src/components/TradeWidget/index.tsx:685]: EDIT: Maybe you are right and the responsability is here?

My thinking now is, the one ignoring the quoted token to get a price is `TradeToken`, so this is the one that gets a price in a different quote to the one user wants to see it in.
Therefore, he makes sure that ask the price always as BUY/SELL, but if SELL is not your your quote, then its his responsibility to invert it.

**Review by anxolin — APPROVED** [review]
It works!!!

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia1.giphy.com%2Fmedia%2Fpslmyy93fH2JboH8qB%2Fgiphy.gif&token=eureka" data-gitmeme-token="eureka"></a>

File: src/components/TradeWidget/index.tsx (hunk 1 above)
Comment (W3stside) [reply src/components/TradeWidget/index.tsx:685]: ok @anxolin did you mean like this https://github.com/gnosis/dex-react/pull/1557/commits/5eb283464db14a1f919b1c3c9753e84ee61dc3fc

Comment (anxolin) [reply src/components/TradeWidget/index.tsx:685]: I was thinking aloud were the responsibility for flipping the price was.
I just didn't want to go back to flipping logic mess. So I did my first comment saying it should be moved to the suggestion

then I edited, and amended my comment, thinking that u may be right, and I explained why I thought so.

So as a summary, I'm not 100% sure, but my current thinking is that u had it right, this is why I approved

Comment (W3stside) [reply src/components/TradeWidget/index.tsx:685]: ah ok, no worries thx

**Pull request merged**